site2/website/release-notes.md

```diff
@@ -1,6 +1,107 @@
 
 ## Apache
 
+### 2.4.2 &mdash; 2019-12-04 <a id="2.4.2"></a>
+
+#### Fixes
+
+* Fixed don't set interrupt flag again after catching interrupt exception in Pulsar Client [#5643](https://github.com/apache/pulsar/pull/5643)
```

Comment:
It's not a clear briefly description for this PR, i think you'd better to check the PR